napari/_qt/qt_viewer.py

```diff
@@ -83,7 +86,7 @@ class QtViewer(QSplitter):
         Button controls for the napari viewer.
     """
 
-    def __init__(self, viewer, welcome=False):
+    def __init__(self, viewer: 'Viewer', welcome=False):
```

Comment:
I believe if you add `from __future__ import annotations`, you do not need the quotes.
Up to you.